griffon/output.py

```diff
@@ -349,11 +346,13 @@ def text_output_products_contain_component(
             console.highlighter = None
 
             flaw_mode = ctx.params["flaw_mode"]
-            flaw_operation = "dry_run"
+            flaw_operation = "new"
             if flaw_mode == "add":
                 flaw_operation = "new"
             if flaw_mode == "update":
                 flaw_operation = "update"
+            if flaw_mode == "dry_run":
+                flaw_operation = "dry_run"
```

Comment:
Nitpick: Maybe some dict with `flaw_mode` to `flaw_operation` mapping would be clearer:
```python
flaw_mode_to_operation = dict(...)
flaw_operation = flaw_mode_to_operation.get(flaw_mode, "new")
```

Reply:
all this affect machinery is tactical ... going to leave asis for now.